src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -166,6 +173,12 @@ private void ConfigureApplication(WebHostBuilderContext context, IApplicationBui
             }
         }
 
+        if (Authentication is WebApplicationAuthenticationBuilder webAuthBuilder && webAuthBuilder.IsAuthenticationConfigured is true)
+        {
+            app.UseAuthentication();
```

Comment:
Do we need to guard this so that the middleware are only added if they're not already added in the app pipeline (like we do with the call to `UseRouting()` above)? Otherwise we'll be forcing auth to run earlier than the app wants to, which can cause problems, e.g. CORS.